gamification/data/config.py

```diff
@@ -76,6 +76,9 @@ def create_badge_activities():
             name='Solved a area/genericbears bear proposal issue'),
         BadgeActivity(
             name='Solved a area/lintbears bear proposal issue'),
+        BadgeActivity(
+            name='Completed Meta-Review'
+        ),
```

Comment:
IMO it's better to keep the format consistent, i.e. put `),` at the end of last line